templates/consoleReadmeSuffix.md

```diff
@@ -0,0 +1,39 @@
+## Getting started
+
+For complete information [see this article](https://docs.apify.com/platform/actors/development#build-actor-at-apify-console). In short, you will:
+
+1. Build the Actor
+2. Run the Actor
+
+## Pull the Actor for local development
+
+In case you would like to move to development on your local machine for customizable developer experience, you can pull the existing Actor from Apify console using Apify CLI:
+
+1. Install `apify-cli`
+
+    **Using Homebrew**
+
+    ```
+    brew install apify/tap/apify-cli
+    ```
+
+    **Using NPM**
+
+    ```
+    npm -g install apify-cli
+    ```
+
+2. Pull the Actor by its unique `<TITLE>`. This command will copy the actor into the current directory on your local machine.
+
+    ```
+    apify pull <TITLE>
```

Comment:
We'd need to parse the md and replace the `<TITLE>` with the actor name in the console. Also, it's not a title, it is a name or ID (e.g. `apify/hello-world` or `E2jjCZBezvAZnX8Rb`)

Reply:
Not sure which script you are referring to but I don't know any that would insert the actor's name into the README. 

Actors created from templates have default names - **my-actor-X** so in theory we can add it into the readme. However, the user can and will most likely rename the actor and we would have to update the README again which we would like to avoid.

So for now we would just keep the generic placeholder `<TITLE>`

Reply:
I know we wanted to make it static to avoid the issues with renaming etc., so it's more for Ondra. I'm fine with keeping static `<ID>` as it's definitely a 90/10 solution here..

Reply:
Hmm I previously thought that the pull command would not work with ID for some reason. If it works with ID then we do not have to worry about the renaming

Reply:
Hmm, true. I still think we can move forward with this as it is such a small detail but it'd actually make sense to dynamically add the ID in the console. Should be easy right?

Reply:
I'm glad we replaced `<TITLE>` with <ID> as Actors don't have a title. But still, the templates are usually the first point of contact for users, so they have no idea where to find the ID and it creates friction. The readme does not even say where to find it.

Please either make it clear where and how the dev should get the ID, or replace it dynamically. In the current state, it's not helpful for new users.

Reply:
The dynamic way is more tempting there is just one tiny "problem". We need the actorId for it which means:
1. Create the actor
2. Use the actorId to generate console readme
3. Update the actor

And we are not sure if the third step is okay because it might be updating the actor stats as well.

On the other hand, if we went with the static way, right now I don't see any easy way to navigate the user to find the actor ID aside from the URL.

I will discuss this today with @mtrunkat but I would be grateful for any suggestions @mnmkng :)

Reply:
So after the discussion with @mtrunkat we will go with the static way. In short, we do not want to put the ID of the actor into the readme in case someone else uses the same source code to create their new actors.